src/main/java/com/onevizion/scmdb/facade/DbScriptFacade.java

```diff
@@ -41,23 +42,39 @@ public void init() {
         scriptsInDir = createScriptsFromFiles(appArguments.isGenDdl() || !appArguments.isOmitChanged());
     }
 
-    public List<SqlScript> getNewScripts() {
+    public List<SqlScript> getNotExecutedScripts() {
         Map<String, SqlScript> savedScripts = sqlScriptDaoOra.readMap();
+        List<SqlScript> newScripts = getLocalScripts().stream()
+                                                      .filter(script -> !savedScripts.containsKey(script.getName()))
+                                                      .collect(Collectors.toList());
+        if (!appArguments.isReadAllFilesContent()) {
+            newScripts.forEach(SqlScript::loadContentFromFile);
+        }
+        return newScripts;
+    }
 
-        scriptsInDir.stream()
-                    .filter(this::isDevScript)
-                    .forEach(script -> logger.info("Dev script [" + script.getName() + "] was ignored"));
-
-        List<SqlScript> newScripts = scriptsInDir.stream()
-                                                 .filter(script -> !isDevScript(script))
-                                                 .filter(script -> !savedScripts.containsKey(script.getName()))
-                                                 .collect(Collectors.toList());
+    public List<SqlScript> getDevelopmentScripts() {
```

Comment:
currently in scmdb `development script` means script without numbers, and it will be always excluded.
![image](https://github.com/Onevizion/scmdb/assets/3282060/72f685ac-b28a-44d0-8960-b6d8bf685350)

So I propose to:
1. rename `isDevScript()` to `isIgnoredScript()`
2. put this check inside `createScriptsFromFiles()`
3. remove `getLocalScripts()` and use `scriptsInDir` since it will no longer contain ignored scripts

Reply:
Done